architecture/networking/assembly_available_router_plugins.adoc

```diff
@@ -0,0 +1,33 @@
+[[architecture-additional-concepts-router-plugins]]
+= Available router plug-ins
+{product-author}
```

Comment:
I'm assuming that the Available Router Plug-ins section will be removed from the Routes topic and incorporated within this topic.

Reply:
I was thinking about it. But now that you've commented, yes, I think it's best and will do it :)

Reply:
Oohh I thought the image was different, but it's the same. Yes, definitely will take out.